src/components/CreateAnalysis/SelectedStudyDetail.tsx

```diff
@@ -23,17 +21,19 @@ const SelectedStudyDetail = () => {
         SeriesInstanceUID: img.SeriesInstanceUID,
         fname: img.fname
       }
-    })
+    });
   }
 
   if (images.length > 0) {
-    const { StudyDescription, StudyInstanceUID, PatientBirthDate, Modality } = images[0];
-    content = (
+    const { StudyDescription, PatientBirthDate, Modality } = images[0];
+    return (
       <div className="detail-select">
         <div className="flex_row">
           <div className="half_width padding_2rem">
             <h1 className="study-title"> {StudyDescription} <span className="outtline-box red-small">{Modality}</span></h1>
-            <p className="color_grey s-large">Study Instance UID#{StudyInstanceUID}</p>
+            <p>
+              {Modality === "CR" ? "Xray scans of suspected patient" : "CT scans of suspected patient"}
```

Comment:
```suggestion
              {Modality === "CR" ? "X-ray scans of patient" : "CT scans of patient"}
```

Reply:
Do keep in mind that the "XRay Scan for possible COVID" comes from the Study Description so it isnt guaranteed to mention "x-ray anywhere". But in any case, I do agree that it is rather redundant to have that text, especially when we also have the red CR rectangle there as well.

Reply:
yup, I agree, I mainly put it there because there used to be a floating message that said `chest scans of suspected patient` and I thought that this was a better alternative.

I'm going to remove it, since I do think that we don't really need this piece of information